README.md

```diff
@@ -68,7 +68,7 @@ By default, the build is targeted against Eclipse Oxygen / 4.7.
 You can explicitly set the `eclipse.target` property to
```

Comment:
Above this line is "By default, the build is target against Eclipse Oxygen / 4.7."

That line should also be changed to "... against Eclipse Photon / 4.8."

Reply:
done